apps/docs-snippets/src/guide/contracts/logs.test.ts

```diff
@@ -20,11 +20,11 @@ describe(__filename, () => {
     const value3 = 'Fuel';
     const value4 = [1, 2, 3];
 
-    const { minGasPrice, maxGasPerTx } = provider.getGasConfig();
+    const { minGasPrice } = provider.getGasConfig();
 
     const { logs } = await contract.functions
       .log_values(value1, value2, value3, value4)
-      .txParams({ gasPrice: minGasPrice, gasLimit: maxGasPerTx })
+      .txParams({ gasPrice: minGasPrice, gasLimit: 10_000 })
```

Comment:
No issue with the code,  just an observation:

If I understood it correctly, this `gasPrice` and `gasLimit` combo is necessary because we have `--min-gas-price 1` assigned to our test node. It seems to me that this is a potential use case for test simplification if we were to run a node for each test, as we'd be able to run the node with `--min-gas-price 0` _by default_ for our tests, but we can have a subset of our tests run their nodes with `--min-gas-price 1` to ensure proper fee calculation. 

This way we'd clear out the boilerplate `const { minGasPrice } = provider.getGasConfig(); ....txParams({gasPrice})...;` out of our tests, and we'd actually be able to go deeper on fee calculation in the relevant tests by setting `--min-gas-price` to various values and verifying that it's properly calculated for each one of them.

Reply:
@nedsalk  Well, not exactly. 

We will always need to inform `gasLimit` when submitting a Script Transaction. When it is not informed, the `gas limit` will be 0. We had many places on the SDK where we were setting the `gasLimit` to the chain value `maxGasPerTx` when the `gasLimit` was not being set.

This approach is not safe because could use more gas than the user intends to and it does not work anymore, since it will always end up in the transaction fails due to `MaxGasExceed`.

Because of this, all of those were removed

For `gasPrice` I am setting to the `minGasPrice` accepted by the chain when it is not being informed. 

This is a safe value to set and (I reckon) will not reflect in any problem for the users.